libs/expo/betterangels/src/lib/screens/NotesHmis/HmisLocation/LocationMapModal/Selected.tsx

```diff
@@ -0,0 +1,170 @@
+import { TargetIcon } from '@monorepo/expo/shared/icons';
```

Comment:
almost line for line of `apps/betterangels/src/app/(private-screens)/add-note/Location/LocationMapModal/Selected.tsx`